website/client/js/services/costumeServices.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+(function(){
+  angular
+    .module('habitrpg')
+    .factory('Costume', costumeFactory);
+
+  costumeFactory.$inject = [
+    'Content'
+  ];
+
+  function costumeFactory(Content) {
+    function animalInfo(name, type) {
+      var tmp = name.split('-');
+      var data = {
+        egg: tmp[0],
+        potion: tmp[1]
+      };
+      data.isBasic = Boolean(Content.hatchingPotions[data.potion]);
```

Comment:
Would that mean that I would also need to invert the if statement in `formatAnimal`?